jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/actions/CodeWhispererLoginLearnMoreAction.kt

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codewhisperer.actions
+
+import com.intellij.ide.BrowserUtil
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.DumbAware
+import software.aws.toolkits.resources.message
+import java.net.URI
+
+class CodeWhispererLoginLearnMoreAction : AnAction(message("aws.settings.learn_more")), DumbAware {
```

Comment:
@crossrac let's put all of `LearnMore` related actions into `CodeWhispererLearnMoreAction`

Reply:
Moved `LoginLearnMore` and `SsoLearnMore` actions to `CodeWhispererLearnMoreAction`. Thanks!